site/static/js/can-be-watched-button.component.js

```diff
@@ -11,32 +18,46 @@ export default class CanBeWatchedButton extends AppComponent {
     this.updateState();
     this.app.messagebus.subscribe('user-signed-in', () => this.updateState());
     this.app.messagebus.subscribe('user-signed-out', () => this.updateState());
-    this.app.messagebus.subscribe('shopping-session-completed', (session) => this.shoppingSessionCompleted(session));
+    this.app.messagebus.subscribe('shopping-session-completed', (session) =>
+      this.shoppingSessionCompleted(session)
+    );
   }
 
-  componentWillUnmount(){
-    this.app.messagebus.unsubscribe('shopping-session-completed', () => this.updateState());
-  };
+  componentWillUnmount() {
+    this.app.messagebus.unsubscribe('shopping-session-completed', () =>
+      this.updateState()
+    );
+  }
 
-  updateState(){
-    if (this.props.slug && this.app.shoppingService.isValidType(this.props.slug)){
-      return this.loadAvailability(this.props.slug).then(() => {
-        return this.setState({ loaded: true });
-      }, utils.noop).then(() => {
-        let e = getComponentElement(this);
-        e.classList.toggle('s72-show', this.state.loaded && this.state.isOwned);
-      }, utils.noop);
+  updateState() {
+    if (
+      this.props.slug &&
+      this.app.shoppingService.isValidType(this.props.slug)
+    ) {
```

Comment:
Are we wrapping because of a linter line length? I'm also not that keen on some of these wraps.

Reply:
This is caused by the prettier `printWidth` rule which is a bit awkward because [it doesn't work the way you might expect a linter line width rule to work](https://prettier.io/docs/en/options.html#print-width).  I've bumped it up to 100 which has fixed a lot of those overkill wrappings.